File: submissions/0001-two-sum/solution.java
public class Solution {
    public int[] twoSum(int[] nums, int target) {
        int inputLength = nums.length;
        for(int i = 0; i < inputLength; i++) {
            for(int j = i+1; j < inputLength; j++) {
                if(nums[i] + nums[j] == target) {
                    return new int[] {i+1, j+1};
                }
            }
        }
        return new int[] {0, 0};
    }
}

File: submissions/0001-two-sum/solution.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        num_positions = {}
        for i, v in enumerate(nums):
            if (target - v) in num_positions:
                return [num_positions[target-v], i]
            
            num_positions[v] = i
        
        
